packages/dds/tree/src/feature-libraries/flex-tree/context.ts

```diff
@@ -87,14 +90,12 @@ export class Context implements FlexTreeHydratedContext, IDisposable {
 	private readonly eventUnregister: (() => void)[];
 	private disposed = false;
 
-	/**
```

Comment:
the portion of these docs not out of date was moved to the preferred location, the rest removed.